M/MariaDB_Connector_C/bundled/patches/0004-Add-ws2_32-to-remoteio-libraries.patch

```diff
@@ -1,15 +0,0 @@
-diff --git a/plugins/io/CMakeLists.txt b/plugins/io/CMakeLists.txt
-index 8c304c9..5b6b0f7 100644
---- a/plugins/io/CMakeLists.txt
-+++ b/plugins/io/CMakeLists.txt
-@@ -10,6 +10,6 @@ IF (WITH_CURL)
-                   DEFAULT DYNAMIC
-                   SOURCES ${CC_SOURCE_DIR}/plugins/io/remote_io.c
-                   INCLUDES ${CURL_INCLUDE_DIR}
--                  LIBRARIES ${CURL_LIBRARIES})
-+                  LIBRARIES ws2_32 ${CURL_LIBRARIES})
-   ENDIF()
```

Comment:
This patch may still be needed for the error you're getting

Reply:
Yes, `select` needs linking to ws2_32: https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-select